src/base/Element.php

```diff
@@ -6180,7 +6181,10 @@ protected function statusFieldHtml(): string
     protected function notesFieldHtml(): string
     {
         // todo: this should accept a $static arg
-        /** @var static|DraftBehavior $this */
+        /**
+         * @var static|DraftBehavior $this
+         * @phpstan-ignore varTag.nativeType
```

Comment:
PHPStan seems to have issues with recognizing behaviors and combining the types. This check is [a v2 addition](https://github.com/phpstan/phpstan-src/commit/a69e3bc2f1e87f6da1e65d7935f1cc36bd5c42fe) so I've opted to ignore it where it causes issues currently instead of a global ignore